hr_timesheet_sheet/views/hr_department_views.xml

```diff
@@ -18,18 +18,18 @@
         <field name="name">hr.department.kanban.inherit</field>
         <field name="model">hr.department</field>
         <field name="inherit_id" ref="hr.hr_department_view_kanban" />
-        <field
-            name="groups_id"
-            eval="[(4,ref('hr_timesheet.group_hr_timesheet_user'))]"
-        />
```

Comment:
Why we need to remove this and add groups in each part?

Reply:
This is something needed in v16. Please check the point "Any view with groups_id..." in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0